autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -107,8 +107,9 @@ function sortSections(sections: SectionSelected[]): SectionSelected[] {
   // maps maintain key insertion order, so add all sections to map and remember order of professors
   const TBASections: SectionSelected[] = [];
   sorted.forEach((section) => {
-    const instructorName = section.section.instructor.name;
-    if (instructorName === 'TBA') {
+    // H stands for honors, R stands for regular
+    const instructorName = section.section.instructor.name + (section.section.honors ? 'H' : 'R');
+    if (instructorName.startsWith('TBA')) {
```

Comment:
Is there a case where checking this instead of equality will make a difference? I don't really see a reason to change it back to how it was seeing as both of these should take pretty much the same amount of time, but curious why you made the change

Reply:
The actual name could either be `TBAH` or `TBAR`

Reply:
Oh yeah that's right, I don't know how I didn't notice the order it's in (change name then check if it's TBA), I'd like if you could change it so it checks if the section is TBA first since there isn't really a reason to change the string first, whenever you do I can reapprove this real quick

Reply:
If I were using the site, I would want to still see honors sections at the top, even if the professor was TBA, so I changed it to behave that way. Lmk if y'all like that